cmd/habitat/data_proxy/data_proxy.go

```diff
@@ -39,6 +43,10 @@ type WriteRequest struct {
 	Data        []byte          `json:"data"`
 }
 
+type WriteResponse struct {
+	Error string `json:"error"`
```

Comment:
We have an error field in the `ResponseWrapper`. If you use the `api.WriteError` and `api.WriteResponse` helpers, this will get handled automatically